.github/workflows/web-crux.yml

```diff
@@ -76,19 +70,19 @@ jobs:
     runs-on: ubuntu-20.04
     needs: prerequirements
     container: docker:latest
-    if: github.ref_name == 'develop' || github.ref_name == 'main'
+    if: github.ref_name == 'develop' || github.ref_name == 'main' || github.ref_type == 'tag'
     defaults:
       run:
         working-directory: ${{ env.WORKING_DIRECTORY }}
     steps:
     - name: Checkout
       uses: actions/checkout@v3
 
-    - name: Download /dist artifact
-      uses: actions/download-artifact@v3
-      with:
-        name: crux-web-dist
-        path: ${{ env.WORKING_DIRECTORY }}/dist
+      # if tag isn't the version setted in package.json will make the job fail
```

Comment:
`set` instead of `setted` ?

Reply:
You're right! Thanks! Also fixed in another pipeline where I used this comment.